katello/hooks/pre/32-install_postgresql.rb

```diff
@@ -0,0 +1,7 @@
+# Working around https://tickets.puppetlabs.com/browse/PUP-2169
+if local_postgresql? && facts[:os][:release][:major] == '7' && !app_value(:upgrade)
+  `rpm -qa | grep rh-postgresql10-postgresql-server-syspaths`
+  unless $?.success?
+    Kafo::Helpers.execute('yum -y install rh-postgresql10-postgresql-server-syspaths')
```

Comment:
I know this is not the `upgrade` path, so the system *should* be clean, but would it be sufficient to install `rh-postgresql10-postgresql-server` and let the installer install `rh-postgresql10-postgresql-server-syspaths` later?